docs/sources/operations/storage/boltdb-shipper.md

```diff
@@ -99,4 +106,25 @@ This problem would be faced even during rollouts which is quite common.
 To avoid this, Loki disables deduplication of index when the replication factor is greater than 1 and `boltdb-shipper` is an active or upcoming index type.
 While using `boltdb-shipper` please avoid configuring WriteDedupe cache since it is used purely for the index deduplication, so it would not be used anyways.
 
+### Compactor
+
+Compactor is a BoltDB Shipper specific service which reduces the index size by deduping the index and merging all the files to a single file per table.
+It is highly recommended running a Compactor since a single Ingester creates 96 files per day which includes a lot of duplicate index entries and querying multiple files per table adds up the overall query latency.
```

Comment:
Copy-edit: 
We recommend running a Compactor since a single Ingester creates 96 files per day which include a lot of duplicate index entries and querying multiple files per table adds up the overall query latency.